node/subsystem-util/src/tests.rs

```diff
@@ -63,6 +63,7 @@ impl JobTrait for FakeCollatorProtocolJob {
 	type Metrics = ();
 
 	const NAME: &'static str = "FakeCollatorProtocolJob";
+	const SUBSYSTEM: &'static str = "fake_collator";
```

Comment:
nit: this is in tests, but still triggers my OCD :)
what @drahnr meant in a previous comment, is that we can derive NAME from SUBSYSTEM (or the other way around) by using `NAME.strip_suffix("-job").unwrap_or(NAME)`
I don't see where there would be multiple NAMEs per SUBSYSTEM and the jobs manager will be rewritten anyway for contextual execution

Reply:
Got it. I'll reuse it.

Reply:
done